ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--
+     Hello! 👋
+     Thank you for filing an issue on PouchDB.
+     This template is optional, but it may help us fix your issue faster.
+-->
+
+### Environment (Node.js/browser/hybrid app/etc.)
+
+(Write here.)
+
+### Browser/platform (Chrome/FF/Safari/Edge/iOS/Android/etc.)
+
+(Write here.)
+
+### Adapter (IndexedDB/WebSQL/LevelDB/etc.)
+
+(Write here.)
+
+### Server (CouchDB/Cloudant/Couchbase/PouchDB Server/etc.)
+
+(Write here.)
+
+### Please describe your issue. Test cases are very helpful!
```

Comment:
Maybe be more explicit what 'test case' means, not so sure it's obvious exactly what we're asking for here?